docs/content/how-tos/rules/FL0076.md

```diff
@@ -0,0 +1,27 @@
+---
+title: FL0076
+category: how-to
+hide_menu: true
+---
+
+# FavourStaticEmptyFields (FL0076)
+
+*Introduced in `0.21.1`*
+
+## Cause
+
+Rule favours static empty fields such as String.Empty, List.Empty and also Array.Empty.
```

Comment:
Please replace this sentence with `Use of immediate string "", empty list [] or empty array [||]`.

Reply:
Already resolved.